### DISCUSSION TITLE
cores argument for Q function

### DISCUSSION THREAD

**Discussion by quirinmanz** [mschubert/clustermq#253] [Q&A] [answered]
Hi,

quick question about the `cores` argument: 
I have been changing the template file to set the default `cpus-per-task` (using SLURM).
Is there an argument in `Q` or `register_dopar_cmq` I missed to set it directly in the function call? That seems to work with `memory`. 

Best,
Quirin

**Comment by mschubert** [comment, marked as answer]
The `memory` argument to `Q` is a shortcut for `template=list(memory=<mem>)` (which only exists for `memory`, but not other template fields).

You can supply it using the `template` list, along with any field that may be present in your template. (Related: #209)
